payment_ippay_ach/models/payment_transaction.py

```diff
@@ -13,21 +14,23 @@
 
 
 class PaymentTansaction(models.Model):
+    """PaymentTansaction."""
+
     _inherit = "payment.transaction"
 
     @api.multi
     def _ippay_ach_s2s_do_payment(self, invoice):
         sequence = self.acquirer_id.journal_id.sequence_id
-        check_number = sequence.next_by_id()
+        check_number = ((sequence.next_by_id()).split('/'))[2]
```

Comment:
I added config instruction to setup the sequence to be a simple number.
With that setup this split operation will error.
Please add code to only split if a '/' is found in the sequence.
Also, get the `[-1]` last element instead of `[2]`

Reply:
yes, I will update it